iphone/Classes/GeolocationModule.m

```diff
@@ -955,7 +956,11 @@ -(void)setPurpose:(NSString *)reason
 
 #pragma mark Geolacation Analytics
 
--(void)fireApplicationAnalyticsIfNeeded:(NSArray *)locations{
+-(void)fireApplicationAnalyticsIfNeeded:(NSArray *)locations
+{
+    if ([AnalyticsModule isEventFltered:@"ti.geo"]) {
+        return;
+    }
     static BOOL analyticsSend = NO;
```

Comment:
I don't know why that's there, it was there already. Let me do some research and find out